Dockerfile

```diff
@@ -4,56 +4,51 @@
 # https://github.com/memsql/memsql-docker-quickstart
 #
 
-# Based on Phusion's baseimage
-FROM phusion/baseimage:0.9.16
-CMD ["/sbin/my_init"]
-
-RUN apt-get update && apt-get install -y \
-    libmysqlclient-dev mysql-client \
-    python-dev python-pip \
-    wget jq build-essential \
-    libcurl4-openssl-dev
+FROM debian:8.4
+MAINTAINER Carl Sverre <carl@memsql.com>
+
+RUN apt-get update && \
+    apt-get install -y \
+        libmysqlclient-dev \
+        mysql-client \
+        curl \
+        jq \
+        python-dev \
+        python-pip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # install useful python packages
 RUN pip install memsql ipython psutil
 
-# configure locale
-RUN locale-gen en_US.UTF-8
-RUN update-locale LANG=en_US.UTF-8
-
-# configure environment
-RUN echo en_US.UTF-8 > /etc/container_environment/LANG
-RUN echo en_US.UTF-8 > /etc/container_environment/LC_ALL
-RUN echo en_US:en > /etc/container_environment/LANGUAGE
-RUN echo docker-quickstart > /etc/container_environment/MEMSQL_OPS_USER_AGENT_SUFFIX
-RUN chmod 755 /etc/container_environment
-RUN chmod 644 /etc/container_environment.sh
+# configure locale for utf-8
+RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+ENV LANG en_US.utf8
 
 # set UTC
 RUN ln -sf /usr/share/zoneinfo/UTC /etc/localtime
 
 # setup directories
-RUN mkdir /memsql /memsql-ops /vol-template
+RUN mkdir /memsql /memsql-ops
 
 # download and install MemSQL Ops
 # then reduce size by symlinking objdir and lib from one install to the other
-ADD setup_ops.sh /tmp/setup_ops.sh
+COPY setup_ops.sh /tmp/setup_ops.sh
```

Comment:
Docker claims COPY is the way to go now.

https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy